src/app/tests/suites/certification/Test_TC_DRLK_2_6.yaml

```
@@ -107,40 +107,38 @@ tests:
           error: INVALID_COMMAND
 
     - label:
-          "Step 5: TH sends Get Holiday Schedule Command to DUT with Invalid
-          HolidayIndex as 15"
+          "Step 5: TH sends GetHolidaySchedule Command to DUT with Invalid
+          HolidayIndex > number_holiday_sch_supported ."
       PICS: DRLK.S.F0b && DRLK.S.C12.Rsp && DRLK.S.C12.Tx
       command: "GetHolidaySchedule"
       arguments:
           values:
               - name: "HolidayIndex"
-                value: 15
+                value: NumberOfHolidaySchedulesSupportedValue + 1
       response:
           values:
-              - name: "HolidayIndex"
-                value: 15
               - name: "Status"
                 value: 133
```

```suggestion
                value: 0x85
```
would be more readable, since spec lists these as hex.

---

@bzbarsky-apple  yaml script is updated as per the suggestion.